docs/Tutorial_VALMOD.ipynb

```diff
@@ -0,0 +1,606 @@
+{
```

Comment:
<p>I see, the substitutions above makes the partial derivatives easier to do! I'd actually REALLY like to see this step-by-step derivation if you don't mind. The current notation is a bit clunky. I think if you performed the partial derivative calculation and then, near the end, set the left side to zero, then it would be pretty clear. I know it's a lot of work though</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
You are definitely correct. I felt the same way but wasn't sure what was the best way to do that. Thanks for the suggestion. So, I will keep (1) and (2). Then, for (3), I first take the derivatives and then, I replace the `partial-derivation notation` with 0.